packages/webapp/src/utils/__tests__/credits-application-validation.spec.js

```diff
@@ -5,10 +5,16 @@ import applicant from '../../__mocks__/applicant'
 import creditsPurchaseConstants from '../credits-purchase-constants.js'
 import constants from '../constants.js'
 
+const populateSessionWithDevelopmentProjectInfo = (session) => {
+  session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_DEVELOPMENT_NAME, 'Anything')
+  session.set(creditsPurchaseConstants.redisKeys.CREDITS_PURCHASE_APPLICATION_REFERENCE, 'BNGCRD-L4XCQ-AIZMO')
```

Comment:
This should be `CREDITS_PURCHASE_PLANNING_APPLICATION_REF` for the planning application reference